src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/UtentiPGPage.java

```diff
@@ -141,7 +141,7 @@ public void clickAddUserButton() {
 
     public void waitLoadAggiungiUtentePage() {
         try {
-            getWebDriverWait(10).withMessage("il bottone indietro Utenti / Aggiungi nuovo utente della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(indietroButton.get(0)));
+            getWebDriverWait(10).withMessage("il bottone indietro Utenti / Aggiungi nuovo utente della pagina aggiungi nuovo utente non è visibile").until(ExpectedConditions.visibilityOf(indietroButton));
```

Comment:
delete try/catch

Reply:
in row 163 use webtool and delete, if possible, try/catch

Reply:
Modificato